.github/workflows/integrate.yml

```diff
@@ -0,0 +1,29 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]        
+
+    steps:
+    - uses: browser-actions/setup-chrome@latest
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
```

Comment:
@petrsvihlik Is it needed to use both `npm i` and `npm ci` ? They serve (more or less) the same goal of installing dependencies (see differences -> https://betterprogramming.pub/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26 ) 

I think that using either of these should be ok, or did you encounter some issues with it?